src/Base/XMLTools.h

```diff
@@ -58,6 +59,23 @@ class BaseExport XMLTools
     static std::unique_ptr<XERCES_CPP_NAMESPACE::XMLTranscoder> transcoder;  // NOLINT
 };
 
+// Helper class for XStrLiteral macro
+// This implementation is almost same as Xerces default memory manager.
+class BaseExport XStrMemoryManager final: public XERCES_CPP_NAMESPACE::MemoryManager

```

Comment:
Lint complains: `expected ';' after top level declarator [clang-diagnostic-error]`

Reply:
The linter is confused by the `BaseExport`` macro.